usb/esp_tinyusb/tusb_msc_storage_spiffs.c

```diff
@@ -0,0 +1,196 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// DESCRIPTION:
+// This file contains the code for accessing the storage medium i.e. SPI Flash.
```

Comment:
Please use "spiflash" or "spi_flash" in the file name, then. "spiffs" is usually associated with SPIFFS file system, which is a particular type of file system on top of SPI flash.

Reply:
done